wdl2cwl/main.py

```diff
@@ -632,6 +632,26 @@ def convert(workflow: str) -> str:
                     unit = ast.task_runtime["memory"][temp + 1 : -1].strip()
                     if input_name in input_names:
                         ram_min = get_ram_min_js(input_name, unit)
+            elif "~{select_first(" in ast.task_runtime["memory"]:
```

Comment:
for processing `select_first` in the context of `task_runtime["memory"]`, please send the results to [`get_ram_min_js()`](https://github.com/common-workflow-lab/wdl-cwl-translator/blob/main/wdl2cwl/main.py#L67)

Reply:
I should treat the `select_first` function as a list of inputs and send the first valid input to `get_ram_min_js()`?

Reply:
no; you should send the javascript snippet to `get_ram_min_js` and modify that function to incorporate it